std/string.d

```diff
@@ -3889,6 +3916,7 @@ auto rightJustifier(Range)(Range r, size_t width, dchar fillChar = ' ')
 
             void popFront()
             {
+                import std.range.primitives : popFront;
```

Comment:
I'm all for using selective imports for the most inner scope that is possible and I agree that it makes the code easier to grasp for the reader. My point was only that range primitives are so common, that you shouldn't need to read the imports list to know where they're coming from. But if it's just a couple of places that I guess it's ok.